doc/changelog.rst

```diff
@@ -3,8 +3,141 @@
+-  Fix Codacy badge (:pull:`2382`) by :user:`bouweandela`
+-  Change 'mean' to 'percentile' in doc strings of preprocessor statistics (:pull:`2327`) by :user:`lukruh`
+-  Fixed typo in doc about weighted statistics (:pull:`2387`) by :user:`schlunma`
+
+Fixes for datasets
+~~~~~~~~~~~~~~~~~~
+
+-  Fixing missing height 2m coordinates in GFDL-CM4 and KIOST-ESM (:pull:`2294`) by :user:`Karen-A-Garcia`
+-  Added fix for wrong units of ``clt`` for CIESM and FIO-ESM-2-0 (:pull:`2330`) by :user:`schlunma`
+-  Cmip6 gfdl_cm4: fix tas height fix to work for concatenated scenarios (:pull:`2332`) by :user:`mwjury`
+-  Cordex GERICS REMO2015  lon differences above 10e-4 (:pull:`2334`) by :user:`mwjury`
+-  Download ICON grid without locking (:pull:`2359`) by :user:`bouweandela`
+-  Added ICON fixes for hfls and hfss (:pull:`2360`) by :user:`diegokam`
+-  Added ICON fix for rtnt (:pull:`2366`) by :user:`diegokam`
+-  Expanded ICON extra facets (:pull:`2379`) by :user:`schlunma`
+-  Add 10m height coordinate to SfcWind GFDL-CM4 instead of 2m height (:pull:`2385`) by :user:`Karen-A-Garcia`
+-  Cordex wrf381p: fix tas,tasmax,tasmin height (:pull:`2333`) by :user:`mwjury`
+-  Several minor fixes needed for marine BGC data. (:pull:`2110`) by :user:`ledm`
+
+Installation
+~~~~~~~~~~~~
+
+-  Updated iris pin to ``iris>=3.6.1`` (:pull:`2286`) by :user:`schlunma`
+-  Pin pandas yet again avoid new ``2.2.1`` as well (:pull:`2353`) by :user:`valeriupredoi`
+-  Update Iris pin to avoid using versions with memory issues (:pull:`2408`) by :user:`chrisbillowsMO`
+-  Pin esmpy <8.6.0 (:pull:`2402`) by :user:`valeriupredoi`
+
+Preprocessor
+~~~~~~~~~~~~
+
+-  New preprocessor: :func:`esmvalcore.preprocessor.local_solar_time` (:pull:`2258`) by :user:`schlunma`
+-  Read derived variables from other MIP tables (:pull:`2256`) by :user:`bouweandela`
+-  Added special unit conversion m -> DU for total column ozone (toz) (:pull:`2270`) by :user:`schlunma`
+-  Allow cubes as input for :func:`esmvalcore.preprocessor.bias` preprocessor (:pull:`2183`) by :user:`schlunma`
+-  Add normalization with statistics to many statistics preprocessors (:pull:`2189`) by :user:`schlunma`
+-  More lazy :func:`esmvalcore.preprocessor.extract_levels` preprocessor function (:pull:`2120`) by :user:`bouweandela`
+-  Adding sfcWind derivation from uas and vas  (:pull:`2242`) by :user:`malininae`
+-  Update interval check in resample_hours (:pull:`2362`) by :user:`axel-lauer`
+-  Broadcast properly ``cell_measures`` when using :func:`esmvalcore.preprocessor.extract_shape` with ``decomposed: True`` (:pull:`2348`) by :user:`sloosvel`
+-  Compute volume from ``cell_area`` if available (:pull:`2318`) by :user:`enekomartinmartinez`
+-  Do not expand wildcards for datasets of derived variables where not all input variables are available (:pull:`2374`) by :user:`schlunma`
+-  Cache regridding weights if possible (:pull:`2344`) by :user:`schlunma`
```

Comment:
This should probably be in performance optimization

Reply:
Done here: 9a79ec4df